src/components/navigation/Navbar.tsx

```diff
@@ -126,7 +147,7 @@ const BoxStyled = styled(Box)<BoxProps>(({ theme }) => ({
 /**
  * Basic navigation bar. Can be used with `NavLinks` and `NavLink` to display a responsive list of links.
  */
-const Navbar = ({ children, logo, ...props }: NavbarProps) => {
+const Navbar = ({ children, logo, linkComponent, ...props }: NavbarProps) => {
   const theme = useTheme();
   let resolvedLogo: ImageColorSchemeSwitchType | null | undefined = null;
```

Comment:
A side note... This won't ever be undefined as we are setting it to null.

Reply:
normal in theme.logos?.normal is typed to `'ImageColorSchemeSwitchType | undefined'`, so it can end up `undefined` with
`resolvedLogo = theme.logos?.normal;`

Reply:
I see.

(Feels like this package is getting more complicated every day... )